tests/utils/test_display.py

```diff
@@ -21,6 +21,8 @@ def test_repeated_show():
 
     disp.show(cube)
 
+    disp.canvas.close()
+
```

Comment:
I pushed a commit. I think this should do the trick. Or was it the intention that `disp.show()` would close the canvas before it returns? cc @FirefoxMetzger

Reply:
Oh, this fixes it locally by not for CI. Will have another later.

Reply:
Yes, I think this was the intention. 

`disp.show()`  calls `run` (the event loop) of the respective backend and blocks until the event loop is brought down. Once that happens, all canvases associated with that event loop should be closed as well, so `disp.show` should indirectly close the canvas.